app/admin/events.rb

```diff
@@ -79,4 +86,15 @@
     end
     active_admin_comments
   end
+
+  sidebar 'Featured Projects', only: :show do
+    table_for event.featured_projects do
+      column :project
+      column 'Actions' do |fp|
+        link_to('View', admin_featured_project_path(fp), { class: 'member_link' }) +
```

Comment:
Line is too long. [84/80]<br>Redundant curly braces around a hash parameter.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.